ag-grid-webcomponent/index.js

```diff
@@ -322,6 +322,23 @@ class AgGrid extends HTMLElement {
         });
         self.dispatchEvent(selectionEvent);
       },
+
+      onModelUpdated: function(params) {
+        let visibleRowIds = []
+        params.api.forEachLeafNode((node) => {
+          if (node.displayed == true && node.id) {
+            visibleRowIds.push(node.id);
+          }
+        })
+
+        const visibleRowIdsUpdated = new CustomEvent("visibleRowIdsUpdated", {
```

Comment:
Maybe the event on the JS side should be named more generic. So we can easier extend the returned values without changing the name in the future.

```js
const visibleRowIdsUpdated = new CustomEvent("modelUpdated", {
  detail: {
      visibleRowIds: visibleRowIds,
      moreInformation: [],
      someModelDetail: details
  }
});